modules/users/tests/server/user.server.model.tests.js

```diff
@@ -153,6 +150,8 @@ describe('User Model Unit Tests:', function () {
   });
 
   after(function (done) {
-    User.remove().exec(done);
+    User.remove().exec(function() {
```

Comment:
you're right but I noticed some odd inconsistency with the after() call failing sometimes and wasn't sure if it is attributed to that or not.
WDYT?